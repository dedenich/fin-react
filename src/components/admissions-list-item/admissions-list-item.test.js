import React from 'react';
import renderer from 'react-test-renderer';
import AdmissionsListItem from './admissions-list-item.jsx';


it(`AdmissionsListItem is rendered correctly`, () => {
  const tree = renderer
  .create(
      <AdmissionsListItem
        isFunding={false}
        comment={`comment`}
        amount={12.0}
        date={`12.09 12.23`}
        isContentShown={false}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
