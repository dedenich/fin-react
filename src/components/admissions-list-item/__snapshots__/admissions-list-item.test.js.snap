// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdmissionsListItem is rendered correctly 1`] = `
<li>
  <div
    className="card-list__ind"
  >
    <svg
      className="card-list__ind"
      height="30"
      width="30"
    >
      <circle
        cx="15"
        cy="15"
        fill="#f5625d"
        r="8"
      />
    </svg>
  </div>
  <div
    className="card-list__comment"
  >
    comment
  </div>
  <div
    className="card-list__summa"
  >
    <span
      className="time"
    >
      12.09 12.23
    </span>
  </div>
</li>
`;
